server/src/main/java/org/elasticsearch/index/IgnoreAbove.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+package org.elasticsearch.index;
+
+import org.elasticsearch.xcontent.XContentString;
+
+import java.util.Objects;
+
+/**
+ * This class models the ignore_above parameter in indices.
+ */
+public class IgnoreAbove {
+
+    public static final IgnoreAbove IGNORE_ABOVE_STANDARD_INDICES = new IgnoreAbove(null, IndexMode.STANDARD, IndexVersion.current());
```

Comment:
Double checking, these constants are only used in tests? I see there is usage in constructors in production code, but the constructors are only used in tests? If so would you be able to write this in a comment?